data_prep_tools.py

```diff
@@ -1,31 +1,78 @@
-from nfldb_tables import NFLFrames
 import pandas as pd
+from factorize import nmf_all_positions
 
-positions_list = ['QB', 'TE', 'RB', 'K', 'WR']
 
 def get_yr_until_wk(year, till_week, nfl_frames):
     '''
-    Input: Integers denoting the year/season and what week to collect data to.
-            Optionally NFLFrames instance (to save cost of building new one)
-    Output:  DataFrame containing player_ids, opponent and fanduel points for season 
-    up until week in question
+    Input:  Int, Int - week to collect data to, NFLFrames
```

Comment:
We should give nfl_frames a default arg of NFLFrames() to simplify testing.

It's obvious what year and till_week are, and that they are necessary.  But unless you are touching the code frequently, it's hard to remember what nfl_frames is doing, or why it's necessary.